src/harness/unittests/tsserverProjectSystem.ts

```diff
@@ -1184,6 +1184,28 @@ namespace ts.projectSystem {
             checkNumberOfInferredProjects(projectService, 0);
         });
 
+        it("external project for dynamic file", () => {
+            const file1 = {

```

Comment:
give the file a dynamic filename

Reply:
The idea with `file1` was not for it to be the dynamic file, but to be the real version of the file. This may not be entirely necessary, I guess. Regardless, we don't actually open this file, we just make the host aware of it.

Reply:
You dont need this at all.. better to remove.

Reply:
Removed.